teaclave_cli/Cargo.toml

```diff
@@ -8,7 +8,6 @@ edition = "2018"
 
 [dependencies]
 mesatee_core = "0.1.0"
-quicli = "0.4.0"
```

Comment:
Should we remove quicli in cargo vendor？

Reply:
That's okay to have more dependencies in `third_party/crates-io`. At least, it does not affect the compilation time. We can cleanup them later.

Reply:
Agree. We can cleanup the third_party crates vendor before the release.